examples/kubernetes/mlflow/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# TODO(crcrpar): Use optuna/optuna:py3.7 instead once it gets fixed.
```

Comment:
A nit but I think this sentence can be elaborated to explain what "fixed" refers to.

Reply:
I should refer "dev" images instead here and I think we can use it now. No need to be "fixed".

Reply:
Considering the maintenance cost, it should be optuna/optuna:py3.7-dev though pulling it feels a bit longer to me. So leave this as is now.